src/main/resources/application.yaml

```
@@ -179,6 +189,9 @@ clients:
   etterlatte:
     uri: http://etterlatte-samordning-vedtak.etterlatte
     scope: api://${CLIENT_ENV}-gcp.etterlatte.etterlatte-samordning-vedtak/.default
+  pdl-fullmakt:
+    uri: https://pdl-fullmakt.prod-fss-pub.nais.io
+    scope: api://prod-fss.pdl.pdl-fullmakt/.default
```

Hvis man har dette så trenger man ingen override i `-dev`-configen
```suggestion
    uri: https://pdl-fullmakt.${CLIENT_ENV}-fss-pub.nais.io
    scope: api://${CLIENT_ENV}-fss.pdl.pdl-fullmakt/.default
```

---

Ah det er sant, glemte å fikse >.>